set.mm

```diff
@@ -85113,6 +85138,76 @@ the first case of his notation (simple exponentiation) and subscript it
     ( cmap co wcel wf cvv wa wb elmapex elmapg syl ibi ) ABCDEFZCBAGZOBHFCHFIOP
     JABCKBCAHHLMN $.
 
+  ${
+    $d A f m $.  $d B f m $.  $d V m $.
+    $( The set exponentiation ` ( B ^m A ) ` is the set of all functions that
+       map from ` A ` to ` B ` .  Generalisation of ~ mapvalg for arbitrary
+       classes as domains instead of sets only.  (Contributed by AV,
+       8-Aug-2024.) $)
+    mapfset $p |- ( B e. V -> { f | f : A --> B } = ( B ^m A ) ) $=
+      ( vm wcel cv wf cab cmap co vex feq1 elab cvv simpr fdmexb mpbiri adantr
+      wa elmapd exbiri pm2.43b elmapi impbid1 syl5bb eqrdv ) BDFZEABCGZHZCIZBAJ
+      KZEGZUKFABUMHZUHUMULFZUJUNCUMELZABUIUMMNUHUNUOUHUNUOUNUHUOUNUNUHTBAUMDOUN
+      UHPUNAOFZUHUNUQUMOFUPABUMQRSUAUBUCUMBAUDUEUFUG $.
+  $}
+
+  ${
+    $d A f $.  $d B f $.
+    $( The set exponentiation ` ( B ^m A ) ` is a subset of the class of
+       functions that map from ` A ` to ` B ` .  (Contributed by AV,
```

Comment:
Phrases like:
* functions that map from A to B
* the set A is a subset of the set B

are very heavy compared to:
* functions from A to B
* (the set) A is included in (the set) B
 
where "the set" can often be omitted, unless it has to be emphasized or it's at the beginning of a sentence.

Reply:
> Phrases like:
> 
>     * functions that map from A to B
> 
>     * the set A is a subset of the set B
> 
> 
> are very heavy compared to:
> 
>     * functions from A to B
> 
>     * (the set) A is included in (the set) B
> 

First item OK, for the second item see comment above.

> where "the set" can often be omitted, unless it has to be emphasized or it's at the beginning of a sentence.

If it is important that a class is a set or a proper class, then "the set" is more often used to distinguish the class from a proper class.

Reply:
> If it is important that a class is a set or a proper class, then "the set" is more often used to distinguish the class from a proper class.

Yes, that's the case I was referring to by "it has to be emphasized".